lib/src/Pos/Web/Server.hs

```diff
@@ -84,24 +91,87 @@ application = do
     server <- servantServer
     return $ serve nodeApi server
 
-serveImpl
-    :: (HasConfiguration, MonadIO m)
-    => m Application
+type TLSServer m =
+    m Application
     -> String
     -> Word16
     -> Maybe TlsParams
     -> Maybe Settings
     -> m ()
-serveImpl app host port mWalletTLSParams mSettings =
+
+type GetTLSSettings =
+    String -> Word16 -> TlsParams -> TLSSettings
```

Comment:
Kinda the same comment here -- Seeing `String -> Word16 -> TlsParams -> TLSSettings` in a signature immediately lets me know that it's a function expecting these things. The alias hides that.

Reply:
Yeah, this was _simply_ to avoid repeating this bit of signature twice in both functions. I tend to think on the moment that alias makes it more readable only to regret it later :/ ...

Reply:
They do make it more readable, but less understandable. Unfortunately the two aren't directly correlated :(